.github/workflows/requirements.txt

```diff
@@ -2,4 +2,4 @@
 #
```

Comment:
I think this whole file should be removed - where is it being used?
I figured it was only used in `lint.yaml`